server/views/pages/tiering-additional-information.njk

```diff
@@ -0,0 +1,555 @@
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p>Sentence Management Design have drafted a set of principles to guide this process, but recognise that further work and engagement on these is required once a
+                        decision has been reached on the methodology to be used
+                    </p>
+                    <br/>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>A PSO should not be responsible for the case of Tier A or B cases. </li>
+                        <li>POs can be responsible for cases of all Tiers (to continuity of relationship) </li>
+                        <li>The Protect axis is more heavily weighted than the Change axis to direct PP resource towards prioritising cases with
+                            the most serious risk of harm. 
+                        </li>
+                        <li> Change axis more heavily weighted for Tiers where Person on Probation s are more likely to demonstrate high responsivity
+                            to undertaking change work 
+                        </li>
+                        <li>Tier A and B cases will need to have the highest resource weighting to ensure public protection requirements are met. </li>
+                        <li>The role boundary between PSO and PO appropriate level work aligns to risk of serious harm to a case and falls at the boundary between Tier C and Tier B.</li>
+                        <li>Tiering calculations should reflect the average annual hours, required by a PP, to meet the policy requirements (e.g. National Standards) and associated mandatory
+                            activity of working on a case. 
+                        </li>
+                        <li>Tier C should have a more equitable split between protecting the public and change work </li>
+                        <li>Tier D with lower risk is more focussed on addressing need and change work </li>
+                        <li>Tiering calculations will occur digitally through specific inputs using set criteria (up to date OASys being encouraged), all associated decisions remain professional
+                            judgement of the SPO (e.g allocations) and probation practitioners (e.g blended supervision arrangements)
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        <br/>
+        <br/>
+
+        <h2 class="govuk-heading-m" id="weight-methodology">Development of weightings methodology</h2>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <p>The Unified Tiering Model (UTM) requires weightings to feed other organisational tools such as the Workload Measurement Tool (WMT).
+                        (Current Kainos WMT will be replaced by MoJ WMT by Dec 2021)
```

Comment:
Pretty sure this never happened 😄  we should remove this line.

Reply:
Ah okay. This was lifted directly from the powerpoint